x-pack/plugins/security_solution/public/detections/components/alerts_kpis/common/components.tsx

```diff
@@ -12,17 +12,23 @@ import { PANEL_HEIGHT, MOBILE_PANEL_HEIGHT } from './config';
 import { useStackByFields } from './hooks';
 import * as i18n from './translations';
 
-export const KpiPanel = styled(EuiPanel)<{ height?: number }>`
+export const KpiPanel = styled(EuiPanel)<{ height?: number; $toggleStatus: boolean }>`
```

Comment:
what does the `$` in the `$toggleStatus` mean? just curiosity

Reply:
This could avoid the prop with `$` passing down to `EuiPanel` and cause warnings (as EuiPanel doesn't accept toggleStatus prop) if I remember correctly